ajax/libs/lawnchair/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "lawnchair",
+  "license": "MIT",
+  "filename": "lawnchair.min.js",
+  "keywords": [
+    "lawnchair",
+    "lawnchairjs",
+    "lawnchair.js",
+    "Gruntfile",
+    "Gruntfile.js",
+    "Grunfilejs"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/brianleroux/lawnchair.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
+        "files": [
+          "**/*"
```

Comment:
It's not a good idea to use wildcard `**/*` in non-`dist`/`build`/`release` type of directory, let's write it's rule.